Octokit/Models/Response/RepositoryInvitation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum InvitationPermissionType
```

Comment:
That's right. But the `RepositoryPermission` enum uses `admin, push, pull` and we need `admin, write, read`. So because of that i created a new enum